ui/libproject/devstart.rb

```diff
@@ -0,0 +1,86 @@
+require "optparse"
+require_relative "utils/common"
+
+def install_dependencies()
+  common = Common.new
+  common.docker.requires_docker
+
+  common.run_inline %W{docker-compose run --rm ui npm install}
+end
+
+def swagger_regen()
+  common = Common.new
+  common.docker.requires_docker
+
+  common.run_inline %W{docker-compose run --rm ui npm run codegen}
+end
+
+class DevUpOptions
+  ENV_CHOICES = %W{local test prod}
+  attr_accessor :env
+
+  def initialize
+    self.env = "test"
```

Comment:
I am happy to have the default environment for the UI be test, but wanted to bring it to other people's attention in case they disagree.